### ISSUE TITLE
deep_svdd_loss Input

### ISSUE THREAD

**Issue by heernink** [zamanzadeh/DACAD#1]
Hi,

Thank you for your great work on DACAD! I was reviewing the implementation and comparing it to the equations in the paper, and I noticed a possible mismatch in the calculation of the classification loss (deep_svdd_loss).

In the current implementation, the source classification loss is calculated as:
`src_cls_loss = self.pred_loss.deep_svdd_loss(q_s_repr, sample_batched_src['label'], center, squared_radius)`

In the current code, `q_s_repr` is used, which is the TCN-extracted feature representation. However, to match the equation in the paper, i am not sure but it shouldn't `pred_s` be used instead.

Could you confirm whether this is the intended?
